openshift-environment-driver/pom.xml

```diff
@@ -54,6 +54,10 @@
       <groupId>org.jboss.pnc</groupId>
       <artifactId>spi</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>build-coordinator</artifactId>
+    </dependency>
 
```

Comment:
I get what you mean. But I'm also not a fan of the non-standard way. I'd rather have everything defined in pnc-config.json rather have it spread between system property and pnc-config.json